tests/MST-parse-test.scm

```diff
@@ -65,8 +65,8 @@
 )
 
 ; Parse the sentences
-(define parse-1 (observe-mst-mode test-str-1 cnt-mode dist-mult #f))
-(define parse-2 (observe-mst-mode test-str-2 cnt-mode dist-mult #f))
+(define parse-1 (observe-mst-mode test-str-1 cnt-mode dist-mult "#f"))
+(define parse-2 (observe-mst-mode test-str-2 cnt-mode dist-mult "#f"))
```

Comment:
Ughh. The quoted string `#f` ? That's going to lead to confusion. If you want to have a string here, change it to `"empty"` or `"none"` or something that doen't collide with a reserved keyword.